pymc3/step_methods/nuts.py

```diff
@@ -0,0 +1,238 @@
+from .quadpotential import quad_potential
```

Comment:
probably because of my refactor of some of these --  `nuts`, `hmc`, `quad_potential` are all in a subfolder (`pymc3/step_methods/hmc/`) now.